OpenRA.Game/FileFormats/WavLoader.cs

```diff
@@ -35,23 +35,20 @@ public enum WaveType { Pcm = 0x1, ImaAdpcm = 0x11 }
 
 		bool LoadSound(Stream s)
 		{
-			var type = s.ReadASCII(4);
```

Comment:
Please augment, don't just remove.  Removing these checks means that the parser will unconditionally treat the file as a wave, and treat bytes 4..8 as a byte count which is then read.  This will then probably crash, as this bogus size can easily be larger than the actual file size.